modules/Bio/Otter/ServerAction/Region.pm

```diff
@@ -947,7 +947,7 @@ sub check_all_supporting_evidences {
     else {
       my $exon_toplevel = $toplevel_exon->transform('toplevel', $toplevel_exon->slice->coord_system->version);
       if ($exon_toplevel) {
-        $toplevel_exon = $exon_toplevel;
+        $exon = $exon_toplevel;
         if ($start_Exon) {
           $transcript->translation->start_Exon($exon_toplevel) if ($start_Exon == $exon);
           $transcript->translation->end_Exon($exon_toplevel) if ($end_Exon == $exon);
```

Comment:
I don't think this works. $start_Exon and $end_Exon have the reference to the objects of the original transcript. $exon_toplevel is a copy of $toplevel_exon and would have a different reference. The means that the start_Exon and end_Exon would never be updated and would point to exons which are not in the transcript
Because you set $exon to be $exon_toplevel the if statement is probably not needed